allennlp/data/vocabulary.py

```diff
@@ -259,6 +260,19 @@ def __init__(
             min_pretrained_embeddings,
         )
 
+    @classmethod
+    def from_pretrained_transformer(
+        cls, model_name: str, namespace: str = "tokens"
+    ) -> "Vocabulary":
+        vocab = cls.empty()
+        from allennlp.common import cached_transformers
+
+        tokenizer = cached_transformers.get_tokenizer(model_name)
+        result = copy_transformer_vocab(tokenizer)
+        vocab._token_to_index[namespace] = result["token_to_index"]
```

Comment:
@epwalsh  I don't know whether  it's safe to replace the whole vocab with a `dict`